server/controllers/resultsController.js

```diff
@@ -140,6 +140,29 @@ class ResultsController {
       return sendError(res, 500, err);
     }
   }
+
+  static async updateResult(req, res) {
+    try {
+      const updateData = res.locals.result;
+      const updatedResult = await Results.update(updateData, {
+        where: {
+          result_uid: req.body.result_uid
```

Comment:
@jamsgra I think we should not be having the result_uid in the req.body object? we should be having it in the req.query or req.params. What do you think?

Reply:
I agree, I've moved `result_uid` & `user_uid` to `req.query`